src/widgets/range-slider/range-slider.js

```diff
@@ -72,15 +70,11 @@ const usage = `Usage:
 rangeSlider({
   container,
   attributeName,
-  [ tooltips=true ],
-  [ templates.{header, footer} ],
```

Comment:
We probably don't want to remove the templates from the usage right? Nor the cssClasses, collapsible or autoHideContainer?